File: Upcoder/Other/Đếm số nguyên tố trong Mảng/LTCB-125.cs
using System;
namespace ĐêmSNT
{
    public class Program
    {
        static void Main(string[]args)
        {
            int count = 0;
            string [] input = Console.ReadLine().Split(' ');
            int n = int.Parse(input[0]);
            int []A = new int[n];
            for (int i = 0; i < n; i++) 
            {
                A[i] = int.Parse(input[i + 1]); 
                if (CheckSNT(A[i]) && A[i] < 100)
                    count++;
            }
            Console.Write(count);            
        }
        static bool CheckSNT(int n)
        {
            if(n < 2)
                return false;
            for(int i = 2; i * i <= n; i++)
                if(n % i == 0)
                    return false;
            return true;
        }
    }
}

File: Upcoder/Other/Đếm số nguyên tố trong Mảng/LTCB-125.py
def CheckSNT(n):
    if n < 2:
        return False
    for i in range(2, int(n ** 0.5) + 1):
        if n % i == 0:
            return False
    return True

def main():
    count = 0
    input_data = input().split()
    n = int(input_data[0])
    A = [int(input_data[i + 1]) for i in range(n)]
    
    for num in A:
        if CheckSNT(num) and num < 100:
            count += 1
    
    print(count)

if __name__ == "__main__":
    main()